Wishlist/wishlist.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Wishlist</title>
+
+    <link rel="stylesheet" href="/Wishlist/wishlist.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="https://garnish-ui.netlify.app/styles.css">
+</head>
+<body>
+
+    <header class="header-nav flex">
+        <ul class="navigation navbar-left flex">
+            <li><a href="" class="nav-link active-link">Home</a></li>
+            <li><a href="" class="nav-link">Products</a></li>
+        </ul>
+
+        <input class="search-bar" type="text" placeholder="Search" />
+
+        <ul class="navigation navbar-right flex">
+            <li>
+                <a href="" class="nav-link"><i class="material-icons"> favorite_border</i></a>
+            </li>
+
+            <li>
+                <a href="" class="nav-link"><i class="material-icons"> account_circle</i></a>
+            </li>
+
+            <li>
+                <a href="" class="nav-link"><i class="material-icons"> add_shopping_cart</i></a>
+            </li>
+        </ul>
+    </header>
+
+    <h1 class="heading-text center-text">My Wishlist(1)</h1>
+
+    <hr>
+
+    <section class="wishlist-main-container">
```

Comment:
maybe instead of section, main tag would be better as only 1 section is used, so main tag feels appropriate

Reply:
Ok , will do the changes. Thank you.